src/openscad.cc

```diff
@@ -1196,3 +1196,22 @@ int main(int argc, char **argv)
 
   return rc;
 }
+
+#ifdef _WIN32
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wdeprecated-declarations"
+static std::wstring_convert<std::codecvt_utf8_utf16<wchar_t>, wchar_t> convert_wchar;
```

Comment:
This might not be the right time to address this, but the codecvt stuff was deprecated in C++17, so ideally we should try to avoid it. We still have some time though, if alternatives are annoying to establish.